### PR TITLE
Week 18 Mongo API

### PR DESCRIPTION
**Pull request by Yennie-Leek**
This week I havent had time to work omn the project as much. 
I did get it to work and could see the data in Local host, but when trying to deploy, its not working. 
Will fix it in the future.

### CODE REVIEW

**Review by idasylven — CHANGES_REQUESTED** [review]
I am quite impressed you actually managed to get half way though this project - I know what a hectic week you had 🥵  And deployment was the hardest part, no doubt about it! 🔄  🔙  🔚  🤯

When you get the chance, these are the requested bullet points necessary to meet the project brief: 

General requirements: 
- Deploy the API to Heroku or similar hosting service.
- Deploy the database using Mongo cloud or similar.

Blue level requirements:
- The API should have at least 2 routes.
- A minimum of one endpoint to return a single result (single element).

Hopefully deployment will be a piece of cake after doing it again this week 🍰 🤞🏻  And creating just one more endpoint returning a single result you will do in no time 😎

**Comment by Yennie-Leek** [comment]
Added more endpoints and got the deployment to work. 
[https://week18mongoapi.herokuapp.com](url)

File: README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I have not been able to spend much time on this project and I have only followed the lectures. 
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://git.heroku.com/mongoapiweek18.git
+
```
Comment (KarolCislowski) [inline README.md:12]: short documentation would be nice 🤓

File: server.js [hunk 2]
```diff
@@ -1,35 +1,60 @@
...
+})
+
+app.get('/netflixData', async (req, res) => {
+  const netflixData = await NetflixData.find()
+  res.json(netflixData)
+})
+
+app.get('/netflixData/:showId', async (req, res) => {
+  const { showId } = req.params
+  const singleShow = await NetflixData.findById(showId)
```
Comment (KarolCislowski) [inline server.js:56]: 👍🏻

**Review by KarolCislowski — APPROVED** [review]
Nice API 👍🏻

**Pull request closed without merging**